### DISCUSSION TITLE
Ocorreu um erro relacionado à rede ou específico da instância ao estabelecer uma conexão com o SQL Server

### DISCUSSION THREAD

**Discussion by joao-sotero** [balta-io/forum#139] [Tira Dúvidas] [answered]
Fiz a criação do container como descrito no artigo passado em aula de sql server no docker, fiz o relacionamento da porta do docker com a porta da maquina e não funciona. O erro abaixo é apresentado no azure data studio.

A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente.

**Comment by MoreiraRamires** [comment]
João, tudo bom?? Qual foi a aula que você fez?? Assim fica mais fácil de ajudarmos.

**Reply by joao-sotero** [reply]
Estou na primeira aula do curso de Fundamentos do SQL Server

**Comment by andrebaltieri** [comment, marked as answer]
Duplicado!
👉 https://github.com/balta-io/forum/discussions/129

**Reply by joao-sotero** [reply]
Ja rodei a soluções apresentadas no artigo e mesmo assim retorna o mesmo erro.
![image](https://user-images.githubusercontent.com/62224433/205695629-cb59db72-847c-4b0a-b846-82e208ef4f82.png)
![image](https://user-images.githubusercontent.com/62224433/205695889-09415c66-97bc-4d3a-98e7-ca6cc41972da.png)

**Reply by andrebaltieri** [reply]
@joao-sotero executou os comandos `dotnet devcert-https --clean e --trust`?

**Reply by joao-sotero** [reply]
@andrebaltieri, rodei sim, mas continua o mesmo erro. Já rodei algumas soluções que achei na internet e não adianta.
![image](https://user-images.githubusercontent.com/62224433/205864757-b1f1d32d-874c-4d08-8756-bec86d68a0e9.png)

**Reply by andrebaltieri** [reply]
Tenta o seguinte:
1 - Para o SQL Server
2 - Para o Docker
3 - Reinicia o Windows
4 - Starta o Docker
5 - Starta o SQL Server
6 - Executa o devcert-https novamente (clean e trust)

Pode ser que tenha ficado alguma config no cache do WSL que esteja impedindo a conexão.

**Reply by joao-sotero** [reply]
Consegui resolver so mudando "localhost" pelo ip "127.0.0.1"

**Reply by andrebaltieri** [reply]
@joao-sotero muito obrigado por compartilhar a solução... vou atualizar o artigo! 👊
